docs/contribute/bors.md

```diff
@@ -60,10 +60,11 @@ more details.)
 
 - Commit messages are taken from the first post of the PR. This is much nicer than mergify /
   Github's default which just combines all commit subject lines into one. We have set things
-  up so that the text after "TO CONTRIBUTORS" is automatically ignored. Note that
-  contributions from other users will have to be added explicitly to the first post with
-  ["Co-authored-by:" trailers](https://github.blog/2018-01-29-commit-together-with-co-authors/);
-  they aren't automatically added like how mergify and Github previously did.
+  up so that the text after the first occurrence of "`---`" is automatically ignored. Contributions
+  from other users should be automatically added as [co-authors](https://github.blog/2018-01-29-commit-together-with-co-authors/),
+  like how mergify and Github previously did. For PRs with a lot of commits (50+) they
```

Comment:
```suggestion
  like how mergify and Github previously did. For PRs with a lot of commits (30+) they
```
The API limit seems to be much lower. :crying_cat_face:

```
$ curl -s https://api.github.com/repos/leanprover-community/mathlib/pulls/1564/commits | jq length
30
```

Reply:
Good to know, thanks!